### PR TITLE
Added External Patient System and fixed up localisations

### PR DESCRIPTION
**Pull request by DaveFIND**

### CODE REVIEW

File: cdx_core/spec/dummy/db/schema.rb [hunk 1]
```diff
@@ -181,7 +181,7 @@
   add_index "audit_updates", ["uuid"], name: "index_audit_updates_on_uuid", using: :btree
 
   create_table "comments", force: :cascade do |t|
-    t.date     "commented_on"
+    t.date     "commented_on",                     default: '2016-06-21'
```
Comment (tripledub) [inline cdx_core/spec/dummy/db/schema.rb]: Shouldn't have a default date here

Comment (tripledub) [inline cdx_core/spec/dummy/db/schema.rb]: Something weird is going on here actually it ghosted it's way into my schema earlier

File: cdx_core/db/migrate/20160928093309_add_external_patient_system_to_patients.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+class AddExternalPatientSystemToPatients < ActiveRecord::Migration
+  def change
+	add_column :patients, :external_patient_system, :string
```
Comment (tripledub) [inline cdx_core/db/migrate/20160928093309_add_external_patient_system_to_patients.rb]: 2 spaces

**Comment by DaveFIND** [comment]
Rebase failure.

Manually recreated the changes into a new branch.

**Pull request closed without merging**